clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/model/GoogleHealthCheck.groovy

```diff
@@ -45,6 +45,12 @@ class GoogleHealthCheck {
     int unhealthyThreshold = GoogleHealthCheck.this.unhealthyThreshold
     int healthyThreshold = GoogleHealthCheck.this.healthyThreshold
 
+    //QUESTION: can I add things to view, is this how to do it?
```

Comment:
The various View classes are used by the Providers to return representations of the underlying resources, without having to return _everything_. Basically a normalized view of the model.